tasks/au/meta/helper2016/au_columns_helper.py

```diff
@@ -0,0 +1,205 @@
+    'G57': ['Occupation by Age by Sex', 'people', 'employment'],
+    'G58': ['Occupation by Hours Worked by Sex', 'people', 'employment'],
+    'G59': ['Method of Travel to Work by Sex', 'people', 'transportation'],
+}
+
+DENOMINATOR_FILTERS = ['LPLFS', 'P_Wife_in_RM_', 'P_Educ_85_years_and_over', 'P_Neg_Nil_incme_85_yrs_ovr',
+                       'P_Tot_85_years_and_over', 'P_Hbn_in_RM', 'P_Ptn_in_DFM', '_OthF', '_CF_C_und15',
+                       '_DS_sd_row', '1PF_C_und15', 'DS_Flat_apart', '1PFNC_und15', 'DS_Other_dwg', 'DS_ns',
+                       'DS_Separate_hse', 'DS_Sept_house', '_CFNC', '_CF', '_1PF', '_DS_sd_rw_tc_h_th',
+                       'P_NS_Tot', '_DS_Flt_apart', '_sd_rw_tce_hs_th', '_DS_over_ns',
+                       'P_0_149_Tot', 'P_150_299_Tot', 'P_300_449_Tot', 'P_450_599_Tot', 'P_600_799_Tot',
+                       'P_800_999_Tot', 'P_1000_1199_Tot', 'P_1200_1399_Tot', 'P_1400_1599_Tot', 'P_1600_1799_Tot',
+                       'P_1800_1999_Tot', 'P_2000_2199_Tot', 'P_2200_2399_Tot', 'P_2400_2599_Tot', 'P_2600_2999_Tot',
+                       'P_3000_3999_Tot', 'P_4000_4999_Tot', 'P_5000over_Tot',
+                       'P_0_299_Tot', 'P_300_449_Tot', 'P_450_599_Tot', 'P_600_799_Tot', 'P_800_999_Tot',
+                       'P_1000_1399_Tot', 'P_1400_1799_Tot', 'P_1800_2399_Tot', 'P_2400_2999_Tot', 'P_3000_3999_Tot',
+                       'P_4000_over_Tot', ]
+DENOMINATOR_ALIASES = {
+    'all': {
+        'P_Y8b_15_19': 'P_Y8b_15_19_yrs',
+        'P_Y8b_20_24': 'P_Y8b_20_24_yrs',
+        'P_Y8b_25_34': 'P_Y8b_25_34_yrs',
+        'P_Y8b_35_44': 'P_Y8b_35_44_yrs',
+        'P_Y8b_45_54': 'P_Y8b_45_54_yrs',
+        'P_Y8b_55_64': 'P_Y8b_55_64_yrs',
+        'P_Y8b_65_74': 'P_Y8b_65_74_yrs',
+        'P_Y8b_75_84': 'P_Y8b_75_84_yrs',
+        'P_Hghst_yr_schl_ns_15_19': 'P_Hghst_yr_schl_ns_15_19_yrs',
+        'P_Hghst_yr_schl_ns_20_24': 'P_Hghst_yr_schl_ns_20_24_yrs',
+        'P_Hghst_yr_schl_ns_25_34': 'P_Hghst_yr_schl_ns_25_34_yrs',
+        'P_Hghst_yr_schl_ns_35_44': 'P_Hghst_yr_schl_ns_35_44_yrs',
+        'P_Hghst_yr_schl_ns_45_54': 'P_Hghst_yr_schl_ns_45_54_yrs',
+        'P_Hghst_yr_schl_ns_55_64': 'P_Hghst_yr_schl_ns_55_64_yrs',
+        'P_Hghst_yr_schl_ns_65_74': 'P_Hghst_yr_schl_ns_65_74_yrs',
+        'P_Hghst_yr_schl_ns_75_84': 'P_Hghst_yr_schl_ns_75_84_yrs',
+        'P_Hghst_yr_schl_ns_85_ovr': 'P_Hghst_yr_schl_ns_85_yrs_ovr',
+    },
+    'G16A': {
+        'P_Tot_15_19': 'P_Tot_15_19_yrs',
+        'P_Tot_20_24': 'P_Tot_20_24_yrs',
+        'P_Tot_25_34': 'P_Tot_25_34_yrs',
+        'P_Tot_35_44': 'P_Tot_35_44_yrs',
+        'P_Tot_45_54': 'P_Tot_45_54_yrs',
+        'P_Tot_55_64': 'P_Tot_55_64_yrs',
+        'P_Tot_65_74': 'P_Tot_65_74_yrs',
+        'P_Tot_75_84': 'P_Tot_75_84_yrs',
+    }
+}
+
+
+def _is_gender_column(short, gender):
+    return short.startswith('{}_'.format(gender)) or short.endswith('_{}'.format(gender))
+
+
+def _is_other_column(short, _):
```

Comment:
Minor: "other"? :thinking: That's very strange :sweat_smile: Can't this be named and implemented after what it _is_, not what it's not? You can probably find something better than "other", even if it has to be "normal_measurement" name or something like that. I guess that "not-based" implementation might be related to the high number of columns. Aren't there patterns that we could match against, instead of negate a few? This kind of "not checks" often lead to surprises when new values appear.

Reply:
This check is true when the measurement is not a `people` one and matches for `income`, `households`, ... I'll try to find a better naming or do a small refactor